input/ig.xml

```diff
@@ -682,7 +682,7 @@
 				<reference value="Specimen/nasoswab"/>
 			</reference>
 			<name value="Specimen - Naso swab (COVID test)"/>
-			<description value="Naso swab specimen for COVID test&#xa;&#xa;Patient: Mr Li Wang"/>
+			<description value="Shows a naso swab specimen referenced by the [COVID Pathology Result (Not detected)](Observation-pathresult-covid-1.html) example for the *AU Core Pathology Result Observation* profile. Patient: Mr. Li Wang."/>
```

Comment:
Title of the example is `[Observation - COVID Pathology Result (Not detected)](https://build.fhir.org/ig/hl7au/au-fhir-core/Observation-pathresult-covid-1.html)`. Please correct

Reply:
Done